cmd/oceanbench/broadcast.go

```diff
@@ -343,6 +343,12 @@ func broadcastHandler(w http.ResponseWriter, r *http.Request) {
 			reportError(w, r, req, "could not save broadcast: %v", err)
 			return
 		}
+		// Ensure that the CheckBroadcast cron exists.
```

Comment:
What's your rationale for putting it here rather than in https://github.com/ausocean/cloud/blob/beeff00faca7ec3738340c685701755d65c5d143/cmd/oceanbench/broadcast.go#L399
?

Reply:
I didn't really have a rationale, but now after consideration, I think it actually makes more sense to be here because I think it's more intuitive that the cron only gets set when you click save. If we put it in the saveBroadcast function it will happen at other points that saveBroadcast is called eg. when generating a token.

Reply:
Yeah, that's a fair point.